distribution/lib/Standard/Table/0.0.0-dev/src/Data/Table.enso

```diff
@@ -1953,8 +1956,14 @@ type Table
                     self.write file format=base_format on_existing_file match_columns on_problems
             _ ->
                 methods = if format == JSON_Format then ["write_table"] else Meta.meta (Meta.type_of format) . methods
-                if methods.contains "write_table" then format.write_table file self on_existing_file match_columns on_problems else
-                    Error.throw (File_Error.Unsupported_Output_Type format Table)
+                if methods.contains "write_table" . not then Error.throw (File_Error.Unsupported_Output_Type format Table) else
+                    effective_existing_behaviour = on_existing_file.get_effective_behavior file
+                    tgt_file = if Context.Output.is_enabled then file else
+                        file.create_dry_run_file copy_original=on_existing_file==Existing_File_Behavior.Append
```

Comment:
I'm just not so sure of operator precedence. I assume its correct but it spawns fear in my eyes.
Maybe thiswould be more readable?

```suggestion
                        should_copy = on_existing_file==Existing_File_Behavior.Append
                        file.create_dry_run_file copy_original=should_copy
```